pages/linux/rdesktop.md

```diff
@@ -0,0 +1,28 @@
+# RDESKTOP
+
+> Remote Desktop Protocal client.
+> It can be used to connect remote windows server or PC using RDP protocal.
+
+- Connect to a remote computer, default port is 3389:
```

Comment:
Use a semicolon (;) instead of a comma (,), or put parenthesis around "default port is 3389".